### DISCUSSION TITLE
Building project "RetryCommand.targets"  Fails

### DISCUSSION THREAD

**Discussion by mdockal** [OrchardCMS/OrchardCore.Commerce#295] [Q&A] [answered]
When I downloaded a fresh copy of OrchardCore.Commerce, I can't get it to build I am getting the following errors:

'pnpm' is not recognized as an internal or external command

error MSB3073: The command "pnpm install --prod --prefer-offline" exited with code 9009.

Done building project "RetryCommand.targets" -- FAILED.

I assume I am missing some VS Extention?

**Comment by 0liver** [comment]
Thanks for trying out this project!

`OrchardCore.Commerce` uses [Lombiq's Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions) project under the hood, which expects a decently recent Node.js version. Please check out the exact requirements here: https://github.com/Lombiq/NodeJs-Extensions#prerequisites.

**Comment by sarahelsaig** [comment, marked as answer]
Hi @mdockal, 
please follow the instructions on the "[Setting up your dev environment](https://github.com/OrchardCMS/OrchardCore.Commerce#setting-up-your-dev-environment)" section of the OrchardCore.Commerce Readme.
